src/Faithlife.Diff/DiffUtility.cs

```diff
@@ -14,57 +14,57 @@ public static class DiffUtility
 		/// Finds the differences between the two lists.
 		/// </summary>
 		/// <typeparam name="T">The type of item in the lists.</typeparam>
-		/// <param name="listFirst">The first list.</param>
-		/// <param name="listSecond">The second list.</param>
+		/// <param name="first">The first list.</param>
+		/// <param name="second">The second list.</param>
 		/// <returns>A sequence of pairs that indicate the differences between the lists.</returns>
 		/// <remarks><para>Each pair identifies a range of items in each list. The items before and after each range
 		/// are the same in both lists; the items in each range are the difference. One of the two ranges can be
 		/// empty, but the index of the range still indicates where the missing items are.</para>
 		/// <para>EqualityComparer&lt;T>.Default is used to compare items.</para></remarks>
-		public static IEnumerable<ValueTuple<IndexRange, IndexRange>> FindDifferences<T>(IList<T> listFirst, IList<T> listSecond)
+		public static IReadOnlyList<(IndexRange FirstRange, IndexRange SecondRange)> FindDifferences<T>(IList<T> first, IList<T> second)
```

Comment:
PascalCase is the preferred convention for tuple members?

Reply:
I figured that might start some debate...

I'm not arguing for a global convention for all tuples, but in the case where they're a return value, it's consistent with .NET property naming in general and with `.Item1` and `.Item2` in particular. It seemed like the natural choice for a return value although it does look odd in the method signature.

Reply:
No objection here. I started with PascalCase, switched to camelCase, but now I'm coming back around...